src/snowflake/cli/_plugins/nativeapp/commands.py

```diff
@@ -457,6 +457,10 @@ def app_events(
         default="",
         help="The name of the consumer account in the organization.",
     ),
+    consumer_app_hash: str = typer.Option(
+        default="",
+        help="The SHA-1 hash of the consumer application name",
```

Comment:
if it's the SHA-1 hash of the consumer application name, don't we have the application name from the project definition file?

Reply:
not if it's installed in a consumer account we don't control (the `--consumer-x` options are meant to be used by the app developer to fetch logs from consumer installations, where some of the info is redacted from the events)

Reply:
got it. lgtm.